src/GeneralStateTestsFiller/stTransactionTest/Opcodes_TransactionInitFiller.json

```diff
@@ -306,7 +369,9 @@
                 ":raw 0x6000600060006000730f572e5295c57f15886f9b263e2f6d2d6c7b5ec6620186a0f45060006000f3", 
                 ":raw 0x6000600060006000730f572e5295c57f15886f9b263e2f6d2d6c7b5ec6612710fa5060006000f3", 
                 ":raw 0x60006000fd60006000f3", 
-                ":raw 0x32ff"
+                ":raw 0x32ff",
+                ":label ef_first_byte :raw 0xef",
```

Comment:
I didn't care to make all possible invalid opcodes here.

And this also can be merged independently, because this behavior is not changing yet.